_posts/2024-12-19-cosmicqc.md

```diff
@@ -0,0 +1,285 @@
+2. The code is available via FigShare, but it was a bit hard to find and parse. Since it is not on GitHub, that also limits the usability since there isn't any environment or dependencies I could find and there isn't a way to create issues and contact the maintainers for assistance. As well, I found that the code I could find was not easy to understand and lacked documentation. I want to make clear, I appreciate that they made the code available and open source which is more than some other papers. My main point is that I don't think this method is feasible to include in my standard workflow.
+
+In conclusion, I want to thank the authors of this paper for their hard work and their important contributions to pushing the field towards developing quality control methodology at the single-cell level for morphology profiles.
+This paper fueled the fire that I had started in myself about wanting to pursue working on a new methodology, one that is simple, to filter out poor quality segmentations from single-cell morphology profiles.
+
+---
+
+## Introduction to coSMicQC
+
+After 2 and half years of being in my position and performing image-based profiling, I have come to realize that I am not perfect (shocking I know! 😱).
+No matter how much I try to manually optimize my segmentation parameters in CellProfiler or when using [Cellpose](https://www.cellpose.org/) or [Stardist](https://stardist.net/), it will not work for all of the single cells in my datasets.
+Now, I am not saying this is a fault of my own, but segmentation errors are a persistent issue and is currently a problem that hasn't been solved to this day.
+
+My point is, you can expect in all of your datasets that it will contain some poorly segmented cells that can be described as technical artifacts.
+You can also expect that debris or smudges could also end up being mis-segmented as cells.
+When working with single-cell data, we do not want these technical artifacts impacting the interpretation of our results.
+Thus, was born this simple idea in my head.
+
+Why not just filter out single cells based on the extracted morphology features?
+
+You wouldn't need to train any model or perform any other pipeline.
+You already have the data in your lap to use, but it is how you use that data is important.
+
+In comes, [coSMicQC](https://github.com/WayScience/coSMicQC), or Single cell Morphology Quality Control!
+
+This new method works by creating conditions, represented by dictionaries.
+A condition is a feature or group of features that will detect a specific type of technical artifact.
```

Comment:
```suggestion
A condition is a feature or group of features that will capture a specific type of technical artifact.
```

Reply:
you define condition, but then what is a "dictionary"?

Reply:
Hmm, I don't think the blog needs to be too technical, so I think I will remove it.